packages/plugin-ext/src/plugin/languages.ts

```diff
@@ -204,11 +204,14 @@ export class LanguagesExtImpl implements LanguagesExt {
     }
 
     private transformDocumentSelector(selector: theia.DocumentSelector): SerializedDocumentFilter[] {
-        if (Array.isArray(selector)) {
+        if (isArray(selector)) {
             return selector.map(sel => this.doTransformDocumentSelector(sel)!);
         }
 
         return [this.doTransformDocumentSelector(selector)!];
+        function isArray(arg: unknown): arg is readonly unknown[] {
```

Comment:
I assume this function exists to make the typing behave? It's not obvious, so warrants a comment. Also, I believe the function is misnamed: from the typing info, it asserts that the argument is a readonly array of any member type, right? If so, the name should be "isReadonlyArray".